samples/jwt-server/src/Dockerfile

```diff
@@ -13,7 +13,7 @@
 #   limitations under the License.
 
 # build a jwt-server binary using the golang container
-FROM golang:1.19 as builder
+FROM golang:1.19.4 as builder
```

Comment:
This change is not needed. The 1.19 tag is shared with the latest 1.19.x tag, currently 1.19.4,

Reply:
thanks for review.
Done it.